string object_name
---
geometry_msgs/Quaternion rotation
